manifests/repo.pp

```diff
@@ -1,5 +1,5 @@
 # powerdns::repo
-class powerdns::repo inherits powerdns {
+class powerdns::repo (Integer $pin_priority = $powerdns::params::pin_priority) inherits powerdns {
```

Comment:
If I'm not mistaken all of our configurations are done in the top scope at the moment. In this case I think it kind of makes sense to have it here.

Just putting this out there, I'm open for your suggestions on this.